src/scene/base.rs

```diff
@@ -639,6 +651,7 @@ impl Visit for Base {
         self.tag.visit("Tag", visitor)?;
         self.physics_binding.visit("PhysicsBinding", visitor)?;
         let _ = self.properties.visit("Properties", visitor);
+        self.frustum_culling.visit("FrustumCulling", visitor)?;
```

Comment:
`let _ = self.frustum_culling.visit("FrustumCulling", visitor);`, do not use `?` operator because it will break backward compatibility with previous versions.